GoodDollar/GoodServer
# (Feature) 468 server should send emails with new design

# Description

Implement design for recovery email, magic link and recovery instructions emails according to design.
Design: https://xd.adobe.com/spec/34f09e36-c7b1-48de-4506-1ef996f8fcd7-42c1/grid

Fixes https://app.zenhub.com/workspaces/gooddollar-5d50833888a83c6880d3e345/issues/gooddollar/gooddapp/468

# How Has This Been Tested?

 - in GoodBootstrap/packages/server/src/server/verification/verificationAPI.js line 272 add development for onlyInEnv middleware.
- in GoodBootstrap/packages/server/src/server/send/sendAPI.js line 75 add development for onlyInEnv middleware.
- in GoodBootstrap/packages/server/src/server/send/sendAPI.js line 102 add development for onlyInEnv middleware.
- restart server
- then go and register a new wallet with some accessible email.
- you should receive 3 emails
- check the content/markup of it

# Checklist:

Screenshots:
<img width="757" alt="1" src="https://user-images.githubusercontent.com/49862004/67674360-fac93380-f984-11e9-9b68-f4a1049f874f.png">
----------------------------------------------------------------------------
<img width="691" alt="2" src="https://user-images.githubusercontent.com/49862004/67674362-fac93380-f984-11e9-8c2b-18e9c543cf23.png">
----------------------------------------------------------------------------
<img width="751" alt="3" src="https://user-images.githubusercontent.com/49862004/67674363-fb61ca00-f984-11e9-9420-6e8663b8de73.png">
----------------------------------------------------------------------------
<img width="707" alt="4" src="https://user-images.githubusercontent.com/49862004/67674364-fb61ca00-f984-11e9-81c3-2db7f34199ba.png">
----------------------------------------------------------------------------
<img width="756" alt="5" src="https://user-images.githubusercontent.com/49862004/67674366-fbfa6080-f984-11e9-8f6a-2b811f2c2138.png">
----------------------------------------------------------------------------
<img width="771" alt="6" src="https://user-images.githubusercontent.com/49862004/67674367-fbfa6080-f984-11e9-8b97-29353678d204.png">
